### PR TITLE
Rfx wam env group sdk

### PR DESCRIPTION
**Pull request by owengc**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -23,17 +23,17 @@
   },
   "scriptPreprocessor": "node_modules/babel-jest",
   "devDependencies": {
-    "@types/jest": "^27.0.2",
-    "@webaudiomodules/api": "^2.0.0-alpha.1",
-    "esbuild": "^0.13.9",
+    "@types/jest": "^27.0.3",
+    "@webaudiomodules/api": "file:../api",
```
Comment (boourns) [inline package.json]: should not merge this

File: src/WamGroup.js [hunk 2]
```diff
@@ -0,0 +1,144 @@
...
+			});
+			this._processors.delete(wam.instanceId);
+		}
+	
+		/**
+		 * @param {string} fromId
+		 * @param {string} toId
+		 * @param {number} [output]
+		 */
+		connectEvents(fromId, toId, output) {
```
Comment (boourns) [inline src/WamGroup.js:72]: What is `output` used for when calling connectEvents?  I understand output when connecting AudioNodes - there can be multiple event outputs on a WAM as well?

**Review by boourns — COMMENTED** [review]
Read the code, left a few comments but overall looks great.

Recommend reviewing with whitespace filtered by adding `?w=1` to the end of the pull request URL:
https://github.com/webaudiomodules/sdk/pull/2/files?w=1

Going to try the branch now in my host

File: src/WamGroup.js (hunk 2 above)
Comment (Fr0stbyteR) [reply src/WamGroup.js:72]: Yes, we leave the possibility to emit different events from outputs.

**Review by boourns — APPROVED** [review]

**Pull request merged**